[2022-10-04 13:29:58,400] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: agg_star_wars.create_agg_table manual__2022-10-04T13:29:57.079523+00:00 [queued]>
[2022-10-04 13:29:58,413] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: agg_star_wars.create_agg_table manual__2022-10-04T13:29:57.079523+00:00 [queued]>
[2022-10-04 13:29:58,414] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-10-04 13:29:58,415] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-10-04 13:29:58,416] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-10-04 13:29:58,431] {taskinstance.py:1259} INFO - Executing <Task(PostgresOperator): create_agg_table> on 2022-10-04 13:29:57.113284+00:00
[2022-10-04 13:29:58,437] {standard_task_runner.py:52} INFO - Started process 3897 to run task
[2022-10-04 13:29:58,441] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'agg_star_wars', 'create_agg_table', 'manual__2022-10-04T13:29:57.079523+00:00', '--job-id', '44', '--raw', '--subdir', 'DAGS_FOLDER/star_wars_agg.py', '--cfg-path', '/tmp/tmpjtgamvoj', '--error-file', '/tmp/tmp7q6b5it3']
[2022-10-04 13:29:58,446] {standard_task_runner.py:77} INFO - Job 44: Subtask create_agg_table
[2022-10-04 13:29:58,506] {logging_mixin.py:109} INFO - Running <TaskInstance: agg_star_wars.create_agg_table manual__2022-10-04T13:29:57.079523+00:00 [running]> on host be1a5b89e146
[2022-10-04 13:29:58,583] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=xxxx@xxxxx.com
AIRFLOW_CTX_DAG_OWNER=Ram
AIRFLOW_CTX_DAG_ID=agg_star_wars
AIRFLOW_CTX_TASK_ID=create_agg_table
AIRFLOW_CTX_EXECUTION_DATE=2022-10-04T13:29:57.113284+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-10-04T13:29:57.079523+00:00
[2022-10-04 13:29:58,595] {base.py:79} INFO - Using connection to: id: postgres_local. Host: host.docker.internal, Port: 5432, Schema: postgres, Login: postgres, Password: ***, extra: {}
[2022-10-04 13:29:58,672] {dbapi.py:225} INFO - Running statement: CREATE TABLE IF NOT EXISTS star_wars_agg_data (
            movie_title TEXT,
            oldest_character TEXT
            ), parameters: None
[2022-10-04 13:29:58,678] {postgres.py:71} INFO - NOTICE:  relation "star_wars_agg_data" already exists, skipping

[2022-10-04 13:29:58,695] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=agg_star_wars, task_id=create_agg_table, execution_date=20221004T132957, start_date=20221004T132958, end_date=20221004T132958
[2022-10-04 13:29:58,739] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-10-04 13:29:58,780] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
